### DISCUSSION TITLE
Enlarge textarea horizontally?

### DISCUSSION THREAD

**Discussion by georgebrianb** [saadeghi/daisyui#487] [Q&A] [answered]
Hi, I'm going through the textarea components, and I cannot find a way to make the textarea enlargeable horizontally. It only works vertically. Could you help with that?

**Comment by saadeghi** [comment, marked as answer]
Use Tailwind CSS `resize` utility:  
https://tailwindcss.com/docs/resize  
Example: https://play.tailwindcss.com/3dYwR9t4JV

**Comment by robsontenorio** [comment]
@saadeghi  Just tried your Playground example and  It seems this trick does not work anymore on Safari. Any workarounds for now?


Works ...
```html
<textarea class="textarea textarea-primary p-2 resize">xxx</textarea>
```

Does not works ... annoying!
```html
<textarea class="textarea textarea-primary p-5 resize">xxx</textarea>
```
